easy-ui-react/src/ModeSwitcher/ModeSwitcher.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useState, createContext, useContext, ReactNode } from "react";
+import { Button, Dialog, DialogTrigger, Popover } from "react-aria-components";
+import ExpandMoreIcon400 from "@easypost/easy-ui-icons/ExpandMore400";
+import { Text } from "../Text";
+import {
+  RadioGroup,
+  RadioGroupItemProps,
+  useRadioGroupContext,
+} from "../RadioGroup";
+import { Icon } from "../Icon";
+import { VerticalStack } from "../VerticalStack";
+import { classNames } from "../utilities/css";
+
+import styles from "./ModeSwitcher.module.scss";
+
+// not the actual component
+
+export function FakeTopLevelForgeModeSwitcher() {
+  const [mode, setMode] = useState("production");
+
+  return (
+    <FakeForgeComponent mode={mode as "production" | "test"}>
+      <ModeSwitcher onModeChange={setMode} />
+    </FakeForgeComponent>
+  );
+}
+
```

Comment:
@stephenjwatkins does this align with how we could potentially use this component? the top level Forge component accepts `mode` as a prop and the dispatch function is passed directly to `ForgeLayout.ModeSwitcher` ?